src/discussions/posts/post-editor/PostEditor.jsx

```diff
@@ -229,22 +244,22 @@ function PostEditor({
             </Form.Group>
             {canSelectCohort
               && (
```

Comment:
nit:
```suggestion
            {canSelectCohort && (
```

Reply:
I think the current style is consistent with the rest of the usages.